### PR TITLE
fix(richtext): paste of several images

### PR DESCRIPTION
**Pull request by Rom1-B**
When we pasted text with several images, only the first image was processed.

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | !24266

### CODE REVIEW

**Review by cedric-anne — APPROVED** [review]
OK for me, but it is a partial fix, as pasting text and images together still not work.

Indeed, pasted content is blocked by our logic, and only images are extracted. Fixing it probably requires to review the whole logic. For instance, it would probably neccessary to insert whole pasted content, then send images to server, then replace update `` tag attributes on server callback.

**Review by cedric-anne — COMMENTED** [review]
I try to find a solution to keep cursor at the end of pasted content.

File: js/fileupload.js [hunk 1]
```diff
@@ -316,42 +316,60 @@ var insertImageInTinyMCE = function(editor, image) {
...
             if (isImageFromPaste(event.content)) {
                 stopEvent(event);
 
                 //extract base64 data
-                var base64 = extractSrcFromImgTag(event.content);
-
-                //transform to blob and insert into editor
-                if (base64.length) {
+                $('<div></div>').append(event.content).find('img').each(function() {
+                    var before = content.split(this.outerHTML)[0];
```
Comment (orthagh) [inline js/fileupload.js:326]: it was hard flor me to read this part, maybe we can do some intermediate steps ? (like assign this to a img var, splitting, etc)

**Comment by cedric-anne** [comment]
I close this one as the @Rom1-B has validated the alternative fix.

**Pull request closed without merging**